### ISSUE TITLE
Stack trace when trying to copy a collection

### ISSUE THREAD

**Issue by palchicz** [SynBioHub/synbiohub#628]
Hello, 

I was just playing around with features on Synbiohub and ran into a stack trace.

**Steps to recreate**
1. Navigated to the "manage submissions page" on synbiohub https://synbiohub.org/manage
2. Identified a collection (an export of the collection can be found here:
[Tryingoutthisthing_collection.xml.zip](https://github.com/SynBioHub/synbiohub/files/2019919/Tryingoutthisthing_collection.xml.zip)) I wanted to copy and clicked the copy button.
3. Updated the ID to "try2", the version to "2", and selected the "Prevent submission" option
4. Hit the copy button

**Expected Behavior**
I would get a new collection that was a copy of the old one with an updated Id and version

**Actual Behavior**
No new copy was created and I got the following stack trace:

```
Error: java.lang.NullPointerException
	at org.sbolstandard.core2.SBOLDocument.removeTopLevel(SBOLDocument.java:4399)
	at org.sbolstandard.core2.SBOLDocument.removeCollection(SBOLDocument.java:479)
	at org.synbiohub.CloneSubmissionJob.execute(CloneSubmissionJob.java:83)
	at org.synbiohub.Main$1.run(Main.java:30)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

    at responseCallbacks.(anonymous function) (/home/deploy/synbiohub/lib/java.js:89:24)
    at DestroyableTransform.javaProcess.stdout.pipe.on (/home/deploy/synbiohub/lib/java.js:51:5)
    at emitOne (events.js:96:13)
    at DestroyableTransform.emit (events.js:188:7)
    at readableAddChunk (/home/deploy/synbiohub/node_modules/readable-stream/lib/_stream_readable.js:211:18)
    at DestroyableTransform.Readable.push (/home/deploy/synbiohub/node_modules/readable-stream/lib/_stream_readable.js:170:10)
    at DestroyableTransform.Transform.push (/home/deploy/synbiohub/node_modules/readable-stream/lib/_stream_transform.js:123:32)
    at DestroyableTransform.write [as _transform] (/home/deploy/synbiohub/node_modules/binary-split/index.js:24:14)
    at DestroyableTransform.Transform._read (/home/deploy/synbiohub/node_modules/readable-stream/lib/_stream_transform.js:159:10)
    at DestroyableTransform.Transform._write (/home/deploy/synbiohub/node_modules/readable-stream/lib/_stream_transform.js:147:83)
    at doWrite (/home/deploy/synbiohub/node_modules/readable-stream/lib/_stream_writable.js:337:64)
    at writeOrBuffer (/home/deploy/synbiohub/node_modules/readable-stream/lib/_stream_writable.js:326:5)
    at DestroyableTransform.Writable.write (/home/deploy/synbiohub/node_modules/readable-stream/lib/_stream_writable.js:263:11)
    at Socket.ondata (_stream_readable.js:555:20)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
```

**Comment by cjmyers** [comment]
I could not reproduce this bug, but when I copy there is no copy shown.  The reason is there is a bug in the copy code in that it does not update the topLevel tags.  I could fix that bug, but I’m second guess the utility of this feature.  I’m not sure it is useful or good to have a copy function for collections.  

Zach: did you have a specific use case for this?  Or were you just trying it out because it was there?

**Comment by cjmyers** [comment]
Ah, I could reproduce if I also change the version, I get this error.  I had only changed the id.

I’m still tempted to remove the feature, but I will fix it if there is a good use case for it.

**Comment by palchicz** [comment]
My goal was to create two divergent collections that had a common base so that I could compare the two as I added new/different designs to each. This was the "just playing around" bit...I had no real motivation besides experimenting with and better understanding the tool. 

The issue was that I had a few different SBOL files on my computer that I had named similarly and I couldn't remember which one was the actual source of the original collection. 

I thought the "copy collection" feature would be a quick mechanism for generating these divergent collections while ensuring they shared a common base. I also wouldn't have to worry about forgetting file names the next time I wanted to do something like this.
